ballerina/client_endpoint.bal

```diff
@@ -268,13 +268,15 @@ public type RetryConfiguration record {|
 # + compression - Specifies the way of handling compression (`accept-encoding`) header
 # + retryConfiguration - Configures the retry functionality
 # + auth - Configurations related to client authentication
+# + maxInboundMessageSize - The maximum message size to be permitted for inbound messages
```

Comment:
This is 4MB, right?. Let's add the default size to the API doc.

Reply:
added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/575/commits/09fd6c5e4f0cf3cddb17f0385abf9e833f5a332d